dlhub_sdk/utils/validation.py

```diff
@@ -36,7 +36,7 @@ def _type_name_to_type(name: str) -> Union[type, Tuple[type]]:
     try:
         return type_table.get(name) or __builtins__[name]  # getattr(__builtins__, name) if __builtins__ is a module
     except KeyError:  # AttributeError if __builtins__ is a module
-        raise ValueError(f"found an unknown type name in servable metadata: {name}")
+        raise ValueError(f"found an unknown type name in servable metadata: {name}") from None
```

Comment:
what happens if you raise from `None`? I am not familiar with this pattern

Reply:
It will `raise` the `ValueError` without mentioning that it was the result of a `KeyError` (the user does not need to know that) docs for exception chaining are [here](https://docs.python.org/3/tutorial/errors.html#exception-chaining) if you're interested.

Reply:
The same behavior will happen without the "from None," (i.e., what we had before), if I understand right

Reply:
From the docs: "Exception chaining happens automatically when an exception is raised inside an [except](https://docs.python.org/3/reference/compound_stmts.html#except) or [finally](https://docs.python.org/3/reference/compound_stmts.html#finally) section. This can be disabled by using from None idiom." Since the `ValueError` is raised in an `except` chaining will happen automatically unless it is disabled (if I understand correctly).

Reply:
Shows that I know. Let's keep yours